client/src/app/game/soundButton.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from "react";
+import css from "./soundButton.module.css";
+
+function SoundButton() {
+
+	const [soundOn, setSoundOn] = useState<boolean>(localStorage.getItem("soundOn") === "on");
```

Comment:
We should probably do !== "off" instead in case the localStorage doesn't exist yet.